java/src/ApiImpl.java

```diff
@@ -15,27 +18,77 @@ public abstract class ApiImpl implements AuthenticatedApi{
     private Vault vault;
 
     public List<Reservation> getMyReservations(int userId){
-        //TODO
-		return Collections.emptyList();
+        return ((List<Object[]>)database.runQuery("select * from reservations where user_id = " + userId)).stream().map(Reservation::map).collect(Collectors.toList());
     }
 
+    @AuthorizedRoles(value = {AuthorizedRoles.Roles.ADMIN})
     public List<Invoice> getUserInvoices(String userId){
```

Comment:
Is this an integer value? If so, it may be better to constrain it to int by using int datatype